readme.md

```diff
@@ -29,7 +29,7 @@
 <br />
 
 * **Simple**: prompts has [no big dependencies](http://npm.anvaka.com/#/view/2d/prompts) nor is it broken into a [dozen](http://npm.anvaka.com/#/view/2d/inquirer) tiny modules that only work well together.
-* **User friendly**: prompt uses layout and colors to create beautiful cli interfaces.
+* ** User-friendly**: prompt uses layout and colours to create beautiful client interfaces.
```

Comment:
Why did you add a space here? Please be careful when updating text.

I also think, that `CLI` is the right term. Please do not mix up [CLI](https://en.wikipedia.org/wiki/Command-line_interface) with [client](https://en.wikipedia.org/wiki/Client_(computing)).

Reply:
In fact _CLI_ stands for _C̲ommand L̲ine I̲nterface_

Reply:
Please revert this change